src/benchmarklib/benchmark_runner.cpp

```diff
@@ -1,5 +1,6 @@
 #include <json.hpp>
 
+#include <boost/range/adaptors.hpp>
 #include <random>
 
```

Comment:
You left vector compression out deliberately, right?

Reply:
I felt that `load <filename> [<tablename> [<encoding> [<compression>]]]` would be too complicated if we don't use it. If we need it in the future, we should revisit how the load command takes its arguments.